app/views/shared/_course_signup_aside.html.haml

```diff
@@ -0,0 +1,38 @@
+-#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+-#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_sac_cas.
+
+%aside.card
+  .card-body
+    %h2.mb-2.card-title=t('.summary_title')
+    .card-text
+      %table.table
+        %tbody
+          %tr
+            %td=t('.summary_course_price')
+            %td.text-end=number_to_currency(1300, unit: 'CHF', format: '%n %u')
+          %tr
+            %td=t('.summary_subsidy')
+            %td.text-end - #{number_to_currency(50, unit: 'CHF', format: '%n %u')}
+        %tfoot
+          %tr
+            %td
+            %td.text-end.fw-bold
+              = t('.summary_total')
```

Comment:
"Total" würde ich wohl trotzdem in die erste Spalte schreiben.

Wenn's geht nur eine Linie zwischen dein einzelnen Positionen und dem Total. Keine zwischen den Positionen, kein nach dem Total.